src/main/scala/cromwell/Main.scala

```diff
@@ -74,11 +74,9 @@ object Main extends App {
 
     import PromiseActor.EnhancedActorRef
 
-    val promise = runner.askNoTimeout(RunWorkflow)
-    waitAndExit(promise, CromwellSystem)
+    waitAndExit(runner.askNoTimeout(RunWorkflow), CromwellSystem)
```

Comment:
Mainly it was because `promise` was a terrible and misleading name for a `Future`.

And I couldn't think of anything really succinct to put there instead so just inlined it.